AdblockPlusSafari/AdblockPlus.m

```diff
@@ -149,4 +205,15 @@ - (NSString *)backgroundSessionConfigurationIdentifier
   return [NSString stringWithFormat:@"%@.AdblockPlusSafari.BackgroundSession", _bundleName];
 }
 
+- (NSString *__nonnull)activeFilterListName
+{
+  if (!self.defaultFilterListEnabled && [self.filterLists[CustomFilterListName] downloaded]) {
+    return CustomFilterListName;
+  } else if (self.acceptableAdsEnabled) {
```

Comment:
Nit: We typically avoid `else` after `return` - it's redundant. But I wouldn't insist. Same goes for the other occurrences I pointed out.